src/app/dashboard/dashboard.tsx

```diff
@@ -30,17 +42,12 @@ export async function Dashboard({
 
   return (
     <>
-      <SummarySidebar
-        total={user.total_budget}
-        remaining={user.remaining_budget}
-        pendingEvents={user.pending_event}
-        plannedEvents={user.planned_event}
-        completedEvents={user.completed_event}
-      />
+      <DashboardSidebar {...user} />
       <main className="w-128">
-        {userBudgets.map((budget: Budget) => (
+        {budgets.map((budget: Budget) => (
           <BudgetDisplay
             key={budget.id}
+            organizer={budget.organizer}
```

Comment:
This is the problematic line.  The problem is `budget` is type `Budget` which has a `user_id` field but not `organizer` field.  The `organizer` field was something I added onto it in lines 23-28.  Basically, if the currently logged-in user is the SGA treasurer, then user the `budget.user_id` to find the name of the user that created that budget so we can display it in the budget card.  If the currently logged-in user is and RSO, then just set the `orangizer=""`, which will cause the budget card to not include a column to display the organizer.

The way I tried doing this was by looping through each budget on line 23, and for each budget, adding an `organizer` field that's either the name of the user or an empty string, depending on who the logged in user is.  But that doesn't work because `budget` is still of type `Budget`, so it causes a type error here.